README.md

```diff
@@ -412,7 +411,7 @@ Other Style Guides
     ```
 
   <a name="arrays--mapping"></a>
-  - [4.6](#arrays--mapping) Use [`Array.from`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from) instead of spread `...` for mapping over iterables, because it avoids creating an intermediate array.
+  - [4.6](#arrays--mapping) Use [`Array.from`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from) instead of spread `...` for mapping over iterables like Sets, Maps or NodeLists, because it avoids creating an intermediate array.
```

Comment:
It wasn't clear that this approach should not be used for regular Arrays

Reply:
But this point [4.6](https://github.com/airbnb/javascript/blob/master/README.md#arrays--mapping) is about conversion of other iterables (except arrays) to arrays. And both examples show that
```
// bad
const baz = [...foo].map(bar);

// good
const baz = Array.from(foo, bar);
```
It doesn't make sense to make new array from other array using Array.from.
We should simply use map for arrays

`const baz = foo.map(bar)`

And [11.1](https://github.com/airbnb/javascript/blob/master/README.md#iterators--nope) clearly tells that.

That's why I wanted to emphasise that such approach should not be used for arrays but for other iterables .

Reply:
It's certainly not the clearest way to map one array to another, but there's nothing wrong with doing it. Imagine an argument that could be an array OR a Set - it's fine to put it in Array.from.